src/Pages/Store/Home/Component/Navbar/Navbar.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable jsx-a11y/alt-text */
+import React, { Component } from "react";
+import "./Navbar.scss";
+
+class Navbar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOotdBorder: false,
+      isStoreBorder: false,
+    };
+  }
+  handleClickOotd = (e) => {
+    e.defaultProps;
```

Comment:
어떤 역할을 하나요?
역할을 한다고 하더라도 아래와 같이 되야 될 것 같습니다
```suggestion
    e.defaultProps();
```

Reply:
이 부분들은 현재님께서 처리해줄 예정입니다 ㅜㅜ
저희가 잠시 navbar를 더해서 본다는게 꼬였습니다 ㅜㅜ

현재님께서 만드셔서 현재님께서 다시 올려주실 예정입니다